streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -549,12 +540,12 @@ void to(final TopicNameExtractor<K, V> topicExtractor,
      *     }
      * }
      * }</pre>
+     * Even if any upstream operation was key-changing, no auto-repartition is triggered.
+     * If repartitioning is required, a call to {@link #through(String)} should be performed before {@code transform()}.
      * <p>
      * Transforming records might result in an internal data redistribution if a key based operator (like an aggregation
      * or join) is applied to the result {@code KStream}.
      * (cf. {@link #transformValues(ValueTransformerSupplier, String...)})
-     * </p>
```

Comment:
JavaDocs is not HTML. No closing `</p>` tag required.

Reply:
Heh, I actually just looked it up, because I was surprised all those other `<p>` elements were not closed... Apparently, even in HTML, you don't _have_ to close `<p>` elements:
> Paragraphs are block-level elements, and notably will automatically close if another block-level element is parsed before the closing \</p\> tag.
> https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p

Sometimes, you just have to stop and marvel at HTML...